src/nets/e2e_asr_attctc_th.py

```diff
@@ -417,6 +508,13 @@ def forward(self, enc_hs_pad, enc_hs_len, dec_z, att_prev, scaling=2.0):
         return c, w
 
 
+def th_accuracy(y_all, pad_target, ignore_label):
+    acc = 0
+    pad_pred = y_all.data.view(pad_target.size(0), pad_target.size(1), y_all.size(1)).max(2)[1]
+    mask = pad_target.data != ignore_label
+    return torch.sum(pad_pred.masked_select(mask) == pad_target.data.masked_select(mask)) / torch.sum(mask)
```

Comment:
Final calculation `torch.sum(pad_pred.masked_select(mask) == pad_target.data.masked_select(mask)) / torch.sum(mask)` is `int / int`, therefore, in the case of python2, acc become = 0. (I confirmed) 

We should cast to float. 
I will show my modified version. 
```
def th_accuracy(y_all, pad_target, ignore_label):
    pad_pred = y_all.data.view(pad_target.size(0), pad_target.size(1), y_all.size(1)).max(2)[1]
    mask = pad_target.data != ignore_label
    numerator = torch.sum(pad_pred.masked_select(mask) == pad_target.data.masked_select(mask))
    denominator = torch.sum(mask)
    return float(numerator) / float(denominator)
```

Reply:
Thanks!